packages/node_modules/@ciscospark/common/src/event-envelope.js

```diff
@@ -0,0 +1,35 @@
+/*!
+ * Copyright (c) 2015-2019 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import {SDK_EVENT} from '@ciscospark/common';
+
+/**
+ * Creates a 'webhook' envelope to wrap Webex Teams events in
+ * @param {object} webex - sdk object
+ * @param {string} resource - resource to create an envelope for
+ * @returns {object} - Returns a promise of an event envelope object
+ */
+export default function createEventEnvelope(webex, resource) {
+  return new Promise((resolve, reject) => {
+    webex.people.get('me')
+      .then((person) => {
+        resolve({
+          createdBy: person.id,
+          orgId: person.orgId,
+          resource,
+          // id -- webhook id concept does not correlate to SDK socket event
+          // name -- webhook name concept does not correlate to SDK socket event
+          // targetUrl -- targetUrl concept does not correlate to SDK socket event
+          // secret -- secret concept does not correlate to SDK socket event
+          ownedBy: SDK_EVENT.TO_APP.OWNER.CREATOR,
```

Comment:
Which I think makes this more understandable: `SDK_EVENT.EXTERNAL.OWNER.CREATOR`.

Reply:
Agree, see commit 41ff1a54e925f04e1588abc4d53b969c05b123cc